sqlmesh/dbt/model.py

```diff
@@ -256,27 +275,44 @@ def model_kind(self, context: DbtContext) -> ModelKind:
 
         incremental_kind_kwargs["on_destructive_change"] = on_destructive_change
         incremental_kind_kwargs["on_additive_change"] = on_additive_change
-        for field in ("forward_only", "auto_restatement_cron"):
-            field_val = getattr(self, field, None)
-            if field_val is None:
-                field_val = self.meta.get(field, None)
-            if field_val is not None:
-                incremental_kind_kwargs[field] = field_val
```

Comment:
This used to add `forward_only` to the `incremental_kind_kwargs` when it should really be part of `incremental_by_kind_kwargs`. This PR fixes that.